processor/src/main.rs

```diff
@@ -390,16 +390,16 @@ async fn handle_coordinator_msg<D: Db, C: Coin, Co: Coordinator>(
         } => {
           let mut block_id = <C::Block as Block<C>>::Id::default();
           block_id.as_mut().copy_from_slice(&context.coin_latest_finalized_block.0);
+          let batches = &context.batches;
 
           let key = <C::Curve as Ciphersuite>::read_G::<&[u8]>(&mut key_vec.as_ref()).unwrap();
 
           // We now have to acknowledge every block for this key up to the acknowledged block
-          let (blocks, outputs) =
-            substrate_mutable.scanner.ack_up_to_block(txn, key, block_id).await;
+          let (_, outputs) = substrate_mutable.scanner.ack_up_to_block(txn, key, block_id).await;
```

Comment:
I believe not needed since we get the acknowledged batches directly from serai anymore. Also that would require extra step of saving block_hash -> batches pair into db and reading etc etc..